get_class_matched_articles.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import re
+import shutil
+import requests
+import sys
+
+class DBpediaSearcher:
+    """
+    Run query to Virtuoso SPARQL endpoint
+    """
+    
+    def _get_dbpedia_url(self, language):
+        return u'http://' + ('' if language == u'en' else language + u'.') + u'dbpedia.org'
+
+    def query_endpoint(self, query, language):
+        """
+        Run a query to Virtuoso SPARQL endpoint with given language.
+        Returns results when success, otherwise error stutus and message
+        """
+        params = {'query': query, 'format': 'json'}
+        r = requests.get(self._get_dbpedia_url(language)+'/sparql', params=params)
+        return r.json()['results']['bindings'] if r.ok else {'id_error': r.status_code, 'message': r.reason}
+
+class ArticleExtractor:
+    """
+    Load Wikipedia articles with specific language and ontology class on DBpedia
+    """
+    
+    def __init__(self, debug=True):
+        self.searcher = DBPediaSearcher()
+        self.debug = debug
+
+    def _get_article_ids(language, ontology_class):
+        query = """\
```

Comment:
This is rather a constant, right? For readability, I would prefer to see it at the beginning of the script, with its name all UPPERCASE